### PR TITLE
Use kernelci package from Docker image

### PR DESCRIPTION
**Pull request by gctucker**
Rather than doing git clones, use the `kernelci` Python package already installed in the Docker image.

### CODE REVIEW

**Comment by gctucker** [comment]
Monitor job tested OK on staging: https://bot.staging.kernelci.org/job/kernel-tree-monitor/1705/

File: jobs/test-runner.jpl [hunk 1]
```diff
@@ -126,16 +117,9 @@ node("docker && test-runner") {
...
-                            kci_core,
-                            params.KCI_CORE_URL, params.KCI_CORE_BRANCH)
-                    },
-                    artifacts: {
-                        getArtifacts(artifacts)
-                    },
-                )
+                artifacts: {
+                    getArtifacts(artifacts)
+                },
```
Comment (nuclearcat) [inline jobs/test-runner.jpl:122]: Probably should be }?

**Review by nuclearcat — CHANGES_REQUESTED** [review]

File: jobs/test-runner.jpl (hunk 1 above)
Comment (gctucker) [reply jobs/test-runner.jpl]: Yes, Jenkins complained about that too.  In fact it just needs to be `getArtifacts(artifacts)`.  Fixed now and tested on staging.

**Comment by gctucker** [comment]
Tested OK on staging https://staging.kernelci.org/test/job/kernelci/branch/staging-mainline/kernel/staging-mainline-20220909.6/

**Pull request merged**